Inventory/Inventory.cs

```diff
@@ -22,11 +22,19 @@ public static void AddProduct(Product product)
             }
             
         }
+
+        private static Product? ProductExist(string name)
```

Comment:
I think RetrieveProduct or GetProduct will be more suitable since here we are trying to get the product